packages/features/ee/teams/components/v2/AddNewTeamMembers.tsx

```diff
@@ -30,10 +30,10 @@ const AddNewTeamMembers = (props: { teamId: number }) => {
   const { t } = useLocale();
   const utils = trpc.useContext();
 
-  const { data: team, isLoading } = trpc.useQuery(["viewer.teams.get", { teamId: props.teamId }]);
+  const { data: team, isLoading } = trpc.useQuery(["viewer.teams.getMembers", { teamId: props.teamId }]);
```

Comment:
We don't need the team info just the members query. Team id is passed down as a prop